README.md

```diff
@@ -185,12 +185,23 @@ The user can also press `esc` to toggle the ability cycle through the options wi
 By default, the select prompt is limited to showing 7 options at a time
 and will paginate lists of options longer than that. This can be changed a number of ways:
 
-```golang
-// as a field on a single select
-prompt := &survey.MultiSelect{..., PageSize: 10}
+#### Select option's description
 
-// or as an option to Ask or AskOne
-survey.AskOne(prompt, &days, survey.WithPageSize(10))
+![Example](img/description-demo.gif)
```

Comment:
I appreciate the demo gif, but it would have been enough to just share it in the body of your PR. I don't think it should be present in the git repository itself since this is a Go library that gets downloaded every time it gets included into a project, and it would be great if we could avoid downloading 85kB of unneeded data.